app-text/texlive_texmf/texlive_texmf-2021.recipe

```diff
@@ -0,0 +1,264 @@
+SUMMARY="A comprehensive TeX system - texmf tree"
+DESCRIPTION="TeX Live is an easy way to get up and running with the TeX \
+document production system. It provides a comprehensive TeX system. It \
+includes all the major TeX-related programs, macro packages, and fonts that \
+are free software, including support for many languages around the world."
+HOMEPAGE="http://tug.org/texlive/"
+COPYRIGHT="1996 by collaboration between the TeX user groups."
+LICENSE="GNU GPL v2"
+REVISION="1"
+fullVersion=20210325
+SOURCE_URI="ftp://tug.org/historic/systems/texlive/$portVersion/install-tl-unx.tar.gz"
+SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-texmf.tar.xz"
+SOURCE_URI_3="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-$fullVersion-extra.tar.xz"
+CHECKSUM_SHA256="74eac0855e1e40c8db4f28b24ef354bd7263c1f76031bdc02b52156b572b7a1d"
+CHECKSUM_SHA256_2="ff12d436c23e99fb30aad55924266104356847eb0238c193e839c150d9670f1c"
+CHECKSUM_SHA256_3="46a3f385d0b30893eec6b39352135d2929ee19a0a81df2441bfcaa9f6c78339c"
+SOURCE_DIR="install-tl-20210324"
+SOURCE_DIR_2="texlive-$fullVersion-texmf"
+SOURCE_DIR_3="texlive-$fullVersion-extra"
+ADDITIONAL_FILES="
+	createSubpackageInfos.pl
+	getPackageEntries.pl
+	subpackageInfos.recipe
+	texlive.profile
+	texlive_postinstall.sh
+	"
+
+POST_INSTALL_SCRIPTS="$relativePostInstallDir/texlive_postinstall.sh"
```

Comment:
this doesn't work as such now. When only texlive_texmf is installed (no collection subpackages), it fails because something is missing apparently. I suppose it should at least be moved to the _basic subpackage. Ideally it should be called whenever one of the runtime subpackages is installed, but only once if multiple are installed at the same time.

However, this needs a bit more advanced logic because there no post-transaction scripts in Haiku (like %posttrans in RPM).